PSReadLine/Prediction.Views.cs

```diff
@@ -501,16 +581,35 @@ private void AggregateSuggestions()
 
                 if (_listItems?.Count > 0)
                 {
-                    _listItemHeight = Math.Min(_listItems.Count, ListMaxCount);
+                    // Initialize the view window position here.
+                    _listViewTop = 0;
+                    _listViewEnd = Math.Min(_listItems.Count, _maxViewHeight);
+                    _listViewHeight = _listViewEnd - _listViewTop;
```

Comment:
isn't `_listViewTop` always zero here?

Reply:
It is. I prefer `_listViewHeight = _listViewEnd - _listViewTop` over `_listViewHeight = _listViewEnd` here to show how the height is calculated.